CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Unreleased
+
+### Fixed
+*  Wrong generic constraint placement in ML-style generic definitions with multiple type parameters [#1868](https://github.com/fsprojects/fantomas/issues/1868)
```

Comment:
Ok, nitpicking here, but notice that you have two spaces after the `*`.
All the others have one.
And please the line with a `.`.

Reply:
Done